src/components/facebookchat.main.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as styles from '!!../utils/less-var-loader!../style/common.less';
+
+class FacebookChat extends React.Component {
+  static loadSDKAsynchronously() {
+    (function loadSdk(d, s, id) {
+      const fjs = d.getElementsByTagName(s)[0];
+      if (d.getElementById(id)) return;
+      const js = d.createElement(s); js.id = id;
+      js.src = 'https://connect.facebook.net/en_US/sdk/xfbml.customerchat.js';
```

Comment:
Ideally we don't want to download the chatbot js if it's not needed. scriptjs will help us through this. 
Take a look at the component here and how we can load third-party JS to the page:

https://github.com/elasticpath/react-pwa-reference-storefront/blob/master/src/components/indirecommendations.main.jsx